git/attribs.go

```diff
@@ -102,11 +105,14 @@ func attrPaths(path, workingDir string) []AttributePath {
 			continue
 		}
 
+		has_filter := strings.Contains(line, "filter=lfs")
```

Comment:
Nit: Go variables should be camel-cased.

Reply:
Ah, yes.  Will fix.